api/src/org/apache/cloudstack/api/command/user/vpn/AddVpnUserCmd.java

```diff
@@ -124,7 +124,7 @@ public void execute() {
                 throw new ServerApiException(ApiErrorCode.INTERNAL_ERROR, "Failed to add vpn user");
             }
         }catch (Exception ex) {
```

Comment:
Here two cases there.
1. apply vpn user failed and returned false. then we throw exception at line 124.
2. apply vpn user got thrown an exception then it is catches.